vnext/Desktop/Modules/WebSocketModule.cpp

```diff
@@ -161,7 +164,10 @@ shared_ptr<IWebSocketResource> WebSocketModule::GetOrCreateWebSocket(int64_t id,
     }
     catch (const winrt::hresult_error& e)
     {
-      string message =  string{"[" + e.code()} + "] " + Utf16ToUtf8(e.message());
+      std::wstringstream ss;
+      ss << L"[" << std::hex << std::showbase << std::setw(8) << static_cast<uint32_t>(e.code()) << L"] " << e.message().c_str();
+      string message{winrt::to_string(ss.str()).c_str()};
```

Comment:
>string message{winrt::to_string(ss.str()).c_str()}; [](http://example.com/codeflow?start=6&length=51)

(nit) even though this is an error case, there seems to be many string conversions. Is this the minimal string conversion logic?

Reply:
This could well be, though the previous implementation already had this problem.
I'll add a task to address this on a subsequent PR, for this has already been merged into the "higher" branches.

Reply:
@NikoAri addressed in #8016.